src/fiskaltrust.ifPOS/v1/de/Models/FinishTransactionResponse.cs

```diff
@@ -6,34 +6,32 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class FinishTransactionResponse
     {
-        [DataMember(Order = 1)]
-        public long TransactionNumber { get; set; }
+        [DataMember(Order = 10)]
+        public ulong TransactionNumber { get; set; }
+
+        [DataMember(Order = 20)]
+        public DateTime StartTransactionTimeStamp { get; set; }
 
-        [DataMember(Order = 2)]
-        public DateTime StartTime { get; set; }
+        [DataMember(Order = 30)]
+        public DateTime TimeStamp { get; set; }
 
-        [DataMember(Order = 3)]
-        public DateTime EndTime { get; set; }
+        [DataMember(Order = 40)]
+        public string TseTimeStampFormat { get; set; }
 
-        [DataMember(Order = 4)]
-        public string CertificateSerialNumber { get; set; }
+        [DataMember(Order = 50)]
+        public string TseSerialNumberOctet { get; set; }
```

Comment:
Not sure about adding `Octet` to this property. Is this requiered? For base64 I think yes, because the caller could decode it, but in this case it makes no sense to decode this.

Reply:
out of description it is the octet representation of the byte-array (sha(public-key)).
therefore i keep it as octet so point out the difference to bitconverter.tostring which would give byte-byte-byte... and to base64 representation.